mlrun/db/httpdb.py

```diff
@@ -140,6 +144,18 @@ def get_base_api_url(self, path: str, version: str = None) -> str:
         url = f"{self.base_url}/{path_prefix}/{path}"
         return url
 
+    def request_with_retry(self, method, url, **kwargs):
+        retry_count = 0
+        while True:
+            try:
+                response = self.session.request(method, url, **kwargs)
+                return response
+            except HTTP_RETRY_EXCEPTIONS as exc:
```

Comment:
I want to be able to disable that feature without disabling the `HTTPAdapter` retry mechanism, I don't have a preference in mind for how to implement it, either different configuration or just a feature flag or maybe other option.